var/spack/repos/builtin/packages/kaks-calculator/package.py

```diff
@@ -0,0 +1,59 @@
+##############################################################################
+# Copyright (c) 2013-2017, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class KaksCalculator(MakefilePackage):
+    """KaKs_Calculator adopts model selection and model averaging to calculate
+       nonsynonymous (Ka) and synonymous (Ks) substitution rates, attempting to
+       include as many features as needed for accurately capturing evolutionary
+       information in protein-coding sequences."""
+
+    homepage = "https://code.google.com/archive/p/kaks-calculator/"
```

Comment:
Google code is dead! Where is the code now?

Reply:
I found a newer version. I'm going to close this and compile 2.0 instead